### ISSUE TITLE
Feature Question spaces in quoted strings

### ISSUE THREAD

**Issue by suckowbiz** [adrienverge/yamllint#389]
Hi,

first I would like to thank you for `yamllint`. I always use it to increase quality for automated CI/CD pipelines in enterprise environments, when any YAML is involved.

Was it ever intended to lint leading and trailing spaces in quoted strings?

Examples

- This one would fail:
```yaml
hello: ' world'
world: 'hello '
yaml: ' lint '
```

- This would succeed
```yaml
hello: 'world'
world: 'hello'
yaml: 'lint'
```

Is this a feature that would make sense or is it not implemented for a reason?

**Comment by adrienverge** [comment]
Hello,

This is an interesting topic.

Such a functionality would mean checking the **content** (values of numbers, strings, etc.), while yamllint is intended to lint the **format** of the YAML representation. If we start to check the *content*, then we can also have rules to validate that numbers are ≥ 42, that strings don't contain the word "fuck", and so on.

I believe content/values validation is application-specific, and another tool should be used for that.

**Comment by suckowbiz** [comment]
Ok, I understand. Thank you for clarification.

I am going to close the issue since the requested topic is out of scope (see above).
